Reply/app/src/main/java/com/materialstudies/reply/ui/home/EmailViewHolder.kt

```diff
@@ -45,26 +44,22 @@ class EmailViewHolder(
         binding.run {
             this.listener = listener
             attachmentRecyclerView.adapter = attachmentAdapter
-            root.background = EmailSwipeActionDrawable(root.context)
+            /*
+                TODO: Alter animation to not populate corners
+                root.background = EmailSwipeActionDrawable(root.context)
+             */
         }
     }
 
     fun bind(email: Email) {
+        val starImageView = binding.emailItemStarImageView
         binding.email = email
         binding.root.isActivated = email.isStarred
-
-        // Set the subject's TextAppearance
-        val textAppearance = binding.subjectTextView.context.themeStyle(
-            if (email.isImportant) {
-                R.attr.textAppearanceHeadline4
-            } else {
-                R.attr.textAppearanceHeadline5
-            }
-        )
-        binding.subjectTextView.setTextAppearanceCompat(
-            binding.subjectTextView.context,
-            textAppearance
-        )
+        starImageView.setOnClickListener { view: View ->
+            email.isStarred = !email.isStarred
+            fillStarImageView(view as ImageView, email.isStarred)
```

Comment:
nit: since you already have the local var reference, you can just pass in `starImageView` here instead of casting `view as ImageView`